From 885696be0cf7cf3b66d62f8233ae5f78a73641b3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Jan=C3=9Fen?= <janssen@mpie.de>
Date: Fri, 22 Oct 2021 13:40:16 -0600
Subject: [PATCH] Lammps compatibility

---
 phonolammps/iofile.py | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/phonolammps/iofile.py b/phonolammps/iofile.py
index ee9d1de..09452f1 100644
--- a/phonolammps/iofile.py
+++ b/phonolammps/iofile.py
@@ -99,15 +99,15 @@ def get_structure_from_lammps(command_list, show_log=False):
     na = lmp.get_natoms()
 
     try:
-        xlo =lmp.extract_global("boxxlo", 1)
-        xhi =lmp.extract_global("boxxhi", 1)
-        ylo =lmp.extract_global("boxylo", 1)
-        yhi =lmp.extract_global("boxyhi", 1)
-        zlo =lmp.extract_global("boxzlo", 1)
-        zhi =lmp.extract_global("boxzhi", 1)
-        xy =lmp.extract_global("xy", 1)
-        yz =lmp.extract_global("yz", 1)
-        xz =lmp.extract_global("xz", 1)
+        xlo =lmp.extract_global("boxxlo")
+        xhi =lmp.extract_global("boxxhi")
+        ylo =lmp.extract_global("boxylo")
+        yhi =lmp.extract_global("boxyhi")
+        zlo =lmp.extract_global("boxzlo")
+        zhi =lmp.extract_global("boxzhi")
+        xy =lmp.extract_global("xy")
+        yz =lmp.extract_global("yz")
+        xz =lmp.extract_global("xz")
 
     except UnboundLocalError:
         boxlo, boxhi, xy, yz, xz, periodicity, box_change = lmp.extract_box()
